packages/core/src/Resource/Change/Task/Helpers/Dependencies/createTaskDependencies.js

```diff
@@ -0,0 +1,24 @@
+import { getAllRelatedResources } from '../../../Relation/Relation';
+import filterNewDependencyTasks from '../filterNewDependencyTasks';
+import findCreateTasksForResources from '../findCreateTasksForResources';
+
+export default function createTaskDependencies(tasks, task) {
+  if (task.type !== 'create' && task.type !== 'update') {
+    return task;
+  }
+
+  const related = getAllRelatedResources(task.payload);
+
+  const newCreateTaskDependencies = filterNewDependencyTasks(
```

Comment:
De naam doet vermoeden dat er ook oldCreateTaskDependencies zijn. In feite zijn alle taken nieuw.
De naam crateTaskDependencies is natuurlijk onhandig. Misschien gewoon dependencies?

Reply:
Aangepast naar filteredDependencies en filterExistingDependencies()